guides/hosting/configurations/observability/monolog.md

```diff
@@ -1,15 +1,15 @@
 ---
 nav:
-  title: Monolog
+  title: Logging
   position: 20
```

Comment:
everything has postition 20 :thinking: 
What order do we want to have? my suggestions would be:
* logging
* profiling
* OpeneTelemetry

going from the most basic to the most advanced